Chap_API_Reserved_Keys.tex

```diff
@@ -431,14 +479,22 @@ \subsection{Process realm attributes}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Node realm keys}
-\label{chap:res:nrealm}
+\label{chap:api_rsvd_keys:nrealm}
+
+Node-related information can be retrieved by requesting a key which defaults
+to the node realm or by including the \refattr{PMIX_NODE_INFO} attribute
+in the \refarg{info} array passed to \refapi{PMIx_Get}.
+The target node defaults to the local node unless a different node is specified
+in the \refarg{info} array using
+either the \refattr{PMIX_HOSTNAME} or \refattr{PMIX_NODEID}.
```

Comment:
We also do support requesting node level info for the node where the provide procID is executing. So it really is that you can specify the node (using hostname or nodeID), or request a node-realm key and specify a procID for the proc executing on the target node.